indico/modules/rb_new/client/js/modules/calendar/Calendar.jsx

```diff
@@ -105,8 +142,21 @@ class Calendar extends React.Component {
                                   activeDate={moment(date)}
                                   onDateChange={setDate}
                                   isLoading={isFetching}
+                                  itemClass={EditableTimelineItem}
+                                  itemProps={{onAddSlot: this.onAddSlot}}
                                   longLabel />
                 </Grid.Row>
+                {bookingRoomId && (
+                    <Preloader checkCached={state => !!roomSelectors.getRoom(state, {roomId: bookingRoomId})}
```

Comment:
The correct check  here would be `roomsSelectors.hasDetails(state, {roomId})`, but you can simply use the `roomPreloader(...)` function here

Reply:
I cannot use `roomPreloader(...)` because this is not a `<Route />`